Library/Formula/boost.rb

```diff
@@ -20,6 +20,10 @@ class Boost < Formula
   option "without-single", "Disable building single-threading variant"
   option "without-static", "Disable building static library variant"
   option "with-mpi", "Build with MPI support"
+  option "with-locale", "Build with locale library"
+  option "with-filesystem", "Build with filesystem library"
+  option "with-program_options", "Build with program_options library"
+  option "with-regex", "Build with regex library"
```

Comment:
Apologies, this is my screw up. I misread the upstream documentation, which implies heavily you have to choose to build the `locale` library [throughout](http://www.boost.org/doc/libs/1_59_0/libs/locale/doc/html/building_boost_locale.html) and I missed the end of the line here:

> boost.locale.posix=on or boost.locale.posix=off Enable or disable support of POSIX backend, it is on by default on Linux and Mac OS X

Does make the `ncmpcpp` initial build failure curious though because it was explicitly failing on finding that library.